app/controllers/schools/participants_controller.rb

```diff
@@ -78,6 +78,10 @@ def update_start_term
     else
       render "schools/participants/edit_start_term"
     end
+
+  rescue ActionController::ParameterMissing
+    flash[:warning] = I18n.t('errors.start_term.blank')
```

Comment:
I think the error message is stored here - https://github.com/DFE-Digital/early-careers-framework/blob/2f4e6538c328b43ffbdd33e7428754cb73a836de/config/locales/en.yml#L77